src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,47 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie: { title,
+  description,
+  imgUrl,
+  imdbUrl } }) => (
```

Comment:
this is a bit of mess and not very clear as could be
```suggestion
export const MovieCard = ({
 // if you have 2-3 properties
  movie: {title, description, imgUrl, imdbUrl },
}) => (

or 

export const MovieCard = ({ 
  movie: { 
    title,
    description,
    imgUrl,
    imdbUrl,
  },
}) => (
```

but such destructuring with several properties from argument is better to do inside component